pkg/kn/config/types.go

```diff
@@ -71,12 +74,17 @@ type ChannelTypeMapping struct {
 	Version string
 }
 
+type Profile struct {
+	Annotations map[string]string `yaml:"annotations"`
+}
```

Comment:
I'd nice if `Labels` map was included here as well.

Reply:
My idea is to have indication to inject to every new Ksvc. It can be captured in `//TODO` comment or new GH issue preferably. Not required to complete the hackathon though.  

```
rofiles:
  istio:
    default: "true"
    annotations:
      sidecar.istio.io/inject: "true"
      sidecar.istio.io/rewriteAppHTTPProbers: "true"
      serving.knative.openshift.io/enablePassthrough: "true"
```